tests/integration/test_apply_checks.py

```diff
@@ -826,266 +1633,8 @@ def test_apply_checks_with_is_unique(ws, spark, set_utc_timezone):
-          col_name: col2
-          limit: 10
-    - criticality: error
-      check:
-        function: is_not_greater_than
-        arguments:
-          col_name: col5
-          limit: 2025-03-01
-    - criticality: error
-      check:
-        function: is_not_greater_than
-        arguments:
-          col_name: col6
-          limit: 2025-03-24 01:00:00
-    - criticality: error
-      check:
-        function: is_not_greater_than
-        arguments:
-          col_name: col3
-          limit: col2 + 10
-    
-    # is_valid_date check
-    - criticality: error
-      check:
-        function: is_valid_date
-        arguments:
-          col_name: col5
-    - criticality: error
-      name: col5_is_not_valid_date2
-      check:
-        function: is_valid_date
-        arguments:
-          col_name: col5
-          date_format: yyyy-MM-dd
-    
-    # is_valid_timestamp check
-    - criticality: error
-      check:
-        function: is_valid_timestamp
-        arguments:
-          col_name: col6
-          timestamp_format: yyyy-MM-dd HH:mm:ss
-    - criticality: error
-      name: col6_is_not_valid_timestamp2
-      check:
-        function: is_valid_timestamp
-        arguments:
-          col_name: col6
-    
-    # is_not_in_future check
-    - criticality: error
-      check:
-        function: is_not_in_future
-        arguments:
-          col_name: col6
-          offset: 86400
-    
-    # is_not_in_near_future check
-    - criticality: error
-      check:
-        function: is_not_in_near_future
-        arguments:
-          col_name: col6
-          offset: 36400
-    
-    # is_older_than_n_days check
-    - criticality: error
-      check:
-        function: is_older_than_n_days
-        arguments:
-          col_name: col5
-          days: 10000
-    
-    # is_older_than_col2_for_n_days check
-    - criticality: error
-      check:
-        function: is_older_than_col2_for_n_days
-        arguments:
-          col_name1: col5
-          col_name2: col6
-          days: 2
-
-    # is_unique check
-    - criticality: error
-      check:
-        function: is_unique
-        arguments:
-          col_name: col1
-    - criticality: error
-      name: col1_is_not_unique2
-      check:
-        function: is_unique
-        arguments:
-          col_name: col1
-          window_spec: window(coalesce(col6, '1970-01-01'), '10 minutes')
-
-    # regex_match check
-    - criticality: error
-      check:
-        function: regex_match
-        arguments:
-          col_name: col2
-          regex: '[0-9]+'
-          negate: false
-    
-    # sql_expression check
-    - criticality: error
-      check:
-        function: sql_expression
-        arguments:
-          expression: col3 >= col2 and col3 <= 10
-          msg: ol3 is less than col2 and col3 is greater than 10
-          name: custom_output_name
-          negate: false
-    """
-    )
+    with open("tests/resources/all_checks.yaml", "r", encoding="utf-8") as f:
```

Comment:
This won't work. Need sth like this: `file_path = Path(__file__).parent.parent / "resources" / "all_checks.yaml"`